### ISSUE TITLE
LUA Error when gain "Army of the Light" Reputation through a World Quest

### ISSUE THREAD

**Issue by fubaWoW** [tvlfrosty/Reputation_Guide#72]
Get an LUA Error then gain Reputation with "Army of the Light""

```LUA
10x ReputationGuide/ReputationGuide.lua:3656: bad argument #6 to 'format' (string expected, got nil)
[string "=[C]"]: in function `format'
[string "@ReputationGuide/ReputationGuide.lua"]:3656: in function `DumpReputationChangesToChat'
[string "@ReputationGuide/ReputationGuide.lua"]:404: in function `REP_OnEvent'
[string "*RetailReputationGuide.xml:11_OnEvent"]:1: in function <[string "*RetailReputationGuide.xml:11_OnEvent"]:1>
[string "=[C]"]: in function `resume'
[string "@ArkInventory/Core/ArkInventory.lua"]:13238: in function `ThreadResume'
[string "*ArkInventory.xml:2549_OnUpdate"]:4: in function <[string "*ArkInventory.xml:2549_OnUpdate"]:1>

Locals:
(*temporary) = "Reputation with %s decreased by %d. (Total: %s%d, Left to %s: %d)"
(*temporary) = "Army of the Light"
(*temporary) = 9650
(*temporary) = ""
(*temporary) = -9650
(*temporary) = nil
(*temporary) = 9973
(*temporary) = "string expected, got nil"

```

**Comment by tvlfrosty** [comment]
@fubaWoW Was this before or after The War Within pre-patch event? And do you remember what world quest it was, or was it just any that gave the error?

**Comment by fubaWoW** [comment]
phu... hard to say right now
it was random that i ran into a worldquest from legion and the error popped, so i reported it :D

i can test it later if i do not forget ^^

**Comment by tvlfrosty** [comment]
I am going to assume it's fixed since the 1.2.2 update. If people still encounter this, please make a new issue and let me know which quest you got it on, so I can test it as well.
